src/renderer/RangeInput.tsx

```diff
@@ -27,112 +20,43 @@ const RangeInput = ({
   isDisabled,
   handleChange,
 }: IRangeInputProps) => {
-  const INTERVAL = 200;
+  // Store a copy of the last value so it isn't lost to the throttle
+  const lastValue = useRef<number | undefined>(undefined);
 
-  const increaseButtonRef = useRef<HTMLDivElement | null>(null);
-  const decreaseButtonRef = useRef<HTMLDivElement | null>(null);
-  const [isIncreasing, setIsIncreasing] = useState(false);
-  const [isDecreasing, setIsDecreasing] = useState(false);
-
-  const handleDeltaChangeGain = useCallback(
-    (isIncrement: boolean) => {
-      if (value < max && isIncrement) {
-        handleChange(value + 1);
-      } else if (value > min && !isIncrement) {
-        handleChange(value - 1);
-      }
-    },
-    [handleChange, max, min, value]
-  );
-
-  // Hooks for continuously increasing/decreasing gain
-  useInterval(
-    () => handleDeltaChangeGain(true),
-    isIncreasing ? INTERVAL : undefined
-  );
-
-  useInterval(
-    () => handleDeltaChangeGain(false),
-    isDecreasing ? INTERVAL : undefined
-  );
-
-  // Handlers for pausing continous change of the gain
-  const stopIncrement = useCallback(() => {
-    setIsIncreasing(false);
-    increaseButtonRef.current?.removeEventListener('mouseleave', stopIncrement);
-  }, []);
-
-  const stopDecrement = useCallback(() => {
-    setIsDecreasing(false);
-    decreaseButtonRef.current?.removeEventListener('mouseleave', stopDecrement);
-  }, []);
-
-  // Handlers for various input types
-  const handleArrowInput = useCallback(
-    (isIncrement: boolean) => {
-      if (isDisabled) {
-        return;
-      }
-      if (isIncrement) {
-        // Manually alter gain once to simulate click
-        handleDeltaChangeGain(true);
-
-        // Begin timer for continous adjustment
-        setIsIncreasing(true);
-        increaseButtonRef.current?.addEventListener(
-          'mouseleave',
-          stopIncrement
-        );
-      } else {
-        // Manually alter gain once to simulate click
-        handleDeltaChangeGain(false);
-
-        // Begin timer for continous adjustment
-        setIsDecreasing(true);
-        decreaseButtonRef.current?.addEventListener(
-          'mouseleave',
-          stopDecrement
-        );
-      }
-    },
-    [handleDeltaChangeGain, isDisabled, stopDecrement, stopIncrement]
-  );
-
-  const handleRangeInput = (e: ChangeEvent<HTMLInputElement>) => {
+  const onRangeInput = (e: ChangeEvent<HTMLInputElement>) => {
     const newValue: number = clamp(parseInt(e.target.value, 10), min, max);
+    lastValue.current = newValue;
     handleChange(newValue);
   };
 
-  // Helper for detecting use of the ENTER key
-  const listenForEnter = (e: KeyboardEvent, handler: () => void) => {
-    if (e.code === 'Enter') {
-      handler();
+  const onMouseUp = () => {
```

Comment:
Technically, this probably doesn't catch the next throttle cycle if someone moved the slider and let go immediately after setting the slider value. Too much work to fix the 1% case though so this is fine for now.